README.md

```diff
@@ -160,11 +160,108 @@ def refund():
     self.refundIndex = ind + 30
 ```
 
-## Installation
 
-Requires Python 3.6 or higher
+# Installation 
+Don't panic if installation fails, Viper is still under development and constant changes. Installation will be much simplified/optimized after a stable version release. 
+
+Take a deep breath and follow, please create an issue if any errors encountered. 
+
+It is **strongly recommended** to install in **a virtual Python environment (normally either `virtualenv` or `venv`)**, so that new packages installed and dependencies built are strictly contained in your viper project and will not alter/affect your other dev environment set-up.
+
+To find out how to set-up virtual environment, more infomation check out: [virtualenv guide](http://python-guide-pt-br.readthedocs.io/en/latest/dev/virtualenvs/). Or follow t
+
+
+- **Ubuntu (16.04 LTS)**
+1. Package update:
+```
+sudo apt-get update 
+sudo apt-get -y upgrade
+```
+
+2. For Viper to run, Python3.6 or later is required, know your python version, if the output is `Python 3.5.2`, then install python3.6, otherwise skip step 3
+```
+python3 -V
+```
+
+3. Install python3.6 and some necessary package (*if you haven't installed the package*)
+```
+wget https://www.python.org/ftp/python/3.6.1/Python-3.6.1.tgz
+tar xfz Python-3.6.1.tgz
+cd Python-3.6.1/
+./configure –prefix /usr/local/lib/python3.6
+sudo make
+sudo make install
+sudo apt-get install build-essential libssl-dev libffi-dev python-dev python3.6-dev python3.6
```

Comment:
Remove packages `python-dev python3.6-dev python3.6` because that's what we're providing with this whole flow
Move `sudo apt-get` before `wget` because if you've never installed `build-essential` before, you won't have `make`